grpc-ballerina/tests/43_nested_record_with_streams.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+
+service NestedMsgService {
+    rpc nestedMsgUnary (google.protobuf.StringValue) returns (NestedMsg);
+    rpc nestedMsgServerStreaming (google.protobuf.StringValue) returns (stream NestedMsg);
+    rpc nestedMsgClientStreaming (stream NestedMsg) returns (google.protobuf.StringValue);
+    rpc nestedMsgBidirectionalStreaming (stream NestedMsg) returns (stream NestedMsg);
```

Comment:
Is it necessary to add the streaming type as a suffix to the function name here(eg: `ServerStreaming`, `BidirectionalStreaming`). The streaming type of the function can be known just by input and output. Shall we have generic function names like `hello`, `bye` etc?

Reply:
It is easy to differentiate this specific use case which covers all the gRPC aspects. I'd prefer these namings rather than other generic names.